### ISSUE TITLE
[WinUI] Top left corner of the TitleBar is currently unclickable when you update to WinAPPSDK 1.4

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#17356]
### Description

The updates to TitleBar in WinAppSDK 1.4  introduced a bug that doesn't obey the specified draggable rectangles for the icon area in the top left corner. 

This should be fixed in WinAppSDK 1.4.2 which means this workaround shouldn't be needed

### Workaround

You should be able to hide the icon with the following code

```C#
	public static class MauiProgram
	{
		public static MauiApp CreateMauiApp()
		{
			var builder = MauiApp.CreateBuilder();
			builder
				.UseMauiApp<App>()
				.ConfigureFonts(fonts =>
				{
					fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
					fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
				})
#if WINDOWS
				.ConfigureLifecycleEvents(events =>
				{
					events.AddWindows(configure =>
					{
						configure.OnWindowCreated((window) =>
						{
							window
								.AppWindow
								.TitleBar
								.IconShowOptions = Microsoft.UI.Windowing.IconShowOptions.HideIconAndSystemMenu;
						});
					});
				})
#endif
				;

#if DEBUG
			builder.Logging.AddDebug();
#endif

			return builder.Build();
		}
	}
```

> VS bug [#1885367](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1885367)

**Comment by mattleibow** [comment]
This may work too:

```cs
builder.ConfigureMauiHandlers(handlers =>
{
#if WINDOWS
    WindowHandler.Mapper.AppendToMapping("FixSystemMenu", (handler, window) => {
        handler.PlatformView.AppWindow.TitleBar.IconShowOptions = Microsoft.UI.Windowing.IconShowOptions.HideIconAndSystemMenu;
    });
#endif
});
```

**Comment by daltzctr** [comment]
> This may work too:
> 
> ```cs
> builder.ConfigureMauiHandlers(handlers =>
> {
> #if WINDOWS
>     WindowHandler.Mapper.AppendToMapping("FixSystemMenu", (handler, window) => {
>         handler.PlatformView.AppWindow.TitleBar.IconShowOptions = Microsoft.UI.Windowing.IconShowOptions.HideIconAndSystemMenu;
>     });
> #endif
> });
> ```

This fixed it in my project.

**Comment by Daltz333** [comment]
Backlog since it's a WinUi3 issue?

**Comment by baaaaif** [comment]
With 1.4.2 the top left corner problem got fixed, but WindowsAppSDK >= 1.4 needs a second workaround in my case.
I think others will need this too:

```
//https://github.com/microsoft/microsoft-ui-xaml/issues/8899 Fix WinAppSDK 1.4 TitleBar Buttons barely visible
WindowHandler.Mapper.AppendToMapping("FixSystemMenu", (handler, window) => {
    handler.PlatformView.AppWindow.TitleBar.ButtonForegroundColor = Application.Current.RequestedTheme switch
    {
        AppTheme.Dark => Microsoft.UI.Colors.White,
        AppTheme.Light => Microsoft.UI.Colors.Black,
        _ => Microsoft.UI.Colors.Black
    }; 
});
```

**Comment by PureWeen** [comment]
This is fixed in 1.4.2 of WinAppSDK

**Comment by PureWeen** [comment]
> With 1.4.2 the top left corner problem got fixed, but WindowsAppSDK >= 1.4 needs a second workaround in my case. I think others will need this too:
> 
> ```
> //https://github.com/microsoft/microsoft-ui-xaml/issues/8899 Fix WinAppSDK 1.4 TitleBar Buttons barely visible
> WindowHandler.Mapper.AppendToMapping("FixSystemMenu", (handler, window) => {
>     handler.PlatformView.AppWindow.TitleBar.ButtonForegroundColor = Application.Current.RequestedTheme switch
>     {
>         AppTheme.Dark => Microsoft.UI.Colors.White,
>         AppTheme.Light => Microsoft.UI.Colors.Black,
>         _ => Microsoft.UI.Colors.Black
>     }; 
> });
> ```

Can you create a new issue?
